x-pack/plugins/maps/server/mvt/get_grid_tile.ts

```diff
@@ -49,20 +54,27 @@ export async function getEsGridTile({
       fields: requestBody.fields,
       runtime_mappings: requestBody.runtime_mappings,
     };
-    const tile = await context.core.elasticsearch.client.asCurrentUser.transport.request(
-      {
-        method: 'GET',
-        path,
-        body,
-      },
-      {
-        signal: abortController.signal,
-        headers: {
-          'Accept-Encoding': 'gzip',
-        },
-        asStream: true,
+
+    const tile = await core.executionContext.withContext(
```

Comment:
Instead of passing in core? could you use context.core, like when making the elasticsearch request?

Reply:
No, it's a naming overlap. `context` is a `RequestHandlerContext`. The `.core` object is not the Core-plugin
 https://github.com/elastic/kibana/blob/b6060544cc1808e2aca3a9611b542ae09b335313/src/core/server/index.ts#L456